M2/Macaulay2/packages/Macaulay2Doc/operators/assignment.m2

```diff
@@ -485,21 +487,16 @@ document {
 	  Consequences => {
 	       { "new local variables ", TT "x", ", ", TT "y", ", ", TT "z", ", ... are created" },
 	       { "the expressions c,d,e,... are assigned to the variables x,y,z,..., respectively, as above." },
-	       { "If the left hand side has more elements than the right hand side, then the extra symbols on the left side are given the value ", TO "null", "." },
-	       { "If the left hand side has fewer elements than the right hand side, then the last symbol on the left hand side is given
-     		    as value a sequence containing the trailing elements of the right hand side." 
-		    },
-	       { "If the right hand side is not a sequence, then it is assigned to the first symbol on the left, and the remaining symbols are assigned the
-		    value ", TO "null", "."
-		    }
+	       { "the number of expressions must match the number of variables." }
```

Comment:
When was this documented behavior removed? It actually seems preferable to me.

Reply:
No idea.  I was surprised to learn about it!

It probably wouldn't be too much trouble to restore it.